makiflow/trainers/nn_render/render_trainer.py

```diff
@@ -316,28 +324,29 @@ def _perform_testing(self, exp_params, save_path, path_to_weights, FPS=25):
     def _plot_values(self, exp_params, save_path):
         print('Prepare to plot values...')
 
-        # Plot product of the neural network
+        # Choose random uv data from tests set
         path = exp_params[ExpField.PATH_TEST_UV][int(np.random.choice(len(exp_params[ExpField.PATH_TEST_UV]), 1))]
-        random_number = int(np.random.choice(len(path + '/*.npy'), 1))
-        uv = np.load(path + '/' + str(random_number) + '.npy').astype(np.float32)
+        random_number = int(np.random.choice(len(os.path.join(path, '*.npy')), 1))
+        uv = np.load(os.path.join(path, str(random_number) + '.npy')).astype(np.float32)
         uv = uv.reshape(1, *uv.shape)
         origin_uv = copy.deepcopy(uv)
 
         for _ in range(exp_params[ExpField.BATCH_SIZE] - 1):
             uv = np.concatenate((origin_uv, uv), 0)
 
-        values = []
+        layer_values = []
 
+        # Plot prediction of the Neural Network at the certain layers
```

Comment:
It is not a `prediction`. Other may get confused by this formulation. Please, use another word.